Android/Google.Android.DataTransport/build.cake

```diff
@@ -0,0 +1,125 @@
+#tool nuget:?package=XamarinComponent
+
+#addin nuget:?package=Cake.XCode
+#addin nuget:?package=Cake.Xamarin.Build
+#addin nuget:?package=Cake.Xamarin
+#addin nuget:?package=Cake.FileHelpers
+
+var TARGET = Argument ("t", Argument ("target", "Default"));
+
+string JAR_URL = "";
+Dictionary<string, string> URLS_ARTIFACT_FILES= new Dictionary<string, string>()
+{
+	{
+		$"https://maven.google.com//com/google/android/datatransport/transport-api/2.0.0/transport-api-2.0.0.aar",
+		$"./externals/android/transport-api-2.0.0.aar"
+	},
+	{
+		$"https://maven.google.com//com/google/android/datatransport/transport-backend-cct/2.0.2/transport-backend-cct-2.0.2.aar",
+		$"./externals/android/transport-backend-cct-2.0.2.aar"
+	},
+	{
+		$"https://maven.google.com//com/google/android/datatransport/transport-runtime/2.0.0/transport-runtime-2.0.0.aar",
+		$"./externals/android/transport-runtime-2.0.0.aar"
+	},
+};
+//string ARTIFACT_FILE = "";
+//string NUGET_VERSION=$"{ARTIFACT_VERSION}";
+
+
+Task ("externals")
+	.Does
+	(
+		() =>
+		{
+			Information("externals ...");
+			EnsureDirectoryExists("./externals/android");
+
+			foreach(KeyValuePair<string, string> URL_ARTIFACT_FILE in URLS_ARTIFACT_FILES)
+			{
+				string URL = URL_ARTIFACT_FILE.Key;
+				string ARTIFACT_FILE = URL_ARTIFACT_FILE.Value;
+
+				Information($"    downloading ...");
+				Information($"                {URL}");
+				Information($"    to ");
+				Information($"                {ARTIFACT_FILE}");
+				if ( ! string.IsNullOrEmpty(URL) && ! FileExists(ARTIFACT_FILE))
+				{
+					DownloadFile (URL, ARTIFACT_FILE);
+				}
+			}
+
+			return;
```

Comment:
`void` methods don't need the `return;` here. And the next task.

Reply:
Hm new sttuff for me.
Seems that that return kills performance...